compiler/passes/scopeResolve.cpp

```diff
@@ -1787,9 +1787,6 @@ static void lookup(BaseAST*       scope,
 
     if (symbols.n == 0) {
       if (scope->getModule()->block == scope) {
-        ModuleSymbol* mod = scope->getModule();
-        lookup(mod->block, name, symbols, alreadyVisited);
-
         if (symbols.n == 0 && getScope(scope)) {
```

Comment:
It's not a big deal but one could also choose to eliminate the test for symbols.n == 0 at new line 1790.
This is redundant with old/new 1788.

I think at this point the choices are

1) Commit as is

2) Make the change, test against release, and commit

3) Add a comment and make the tweak the next time you're in this function (on the assumption
that lookup will benefit from a meaningful overhaul at some point).

Mike